packages/react/src/components/UIShell/components/SideNavMenu.tsx

```diff
@@ -73,6 +94,11 @@ export interface SideNavMenuProps {
    */
   isSideNavExpanded?: boolean;
 
+  /**
+   *  Specifices if it's the primary sidenav
```

Comment:
```suggestion
   * Specifies if this is the primary SideNav.
   * If true, child components will open to the right,
   * creating the double-wide navigation layout
```

Adding some more comments here